src/tokens/evo-data-viz.css

```diff
@@ -0,0 +1,10 @@
+:root {
+    --color-data-viz-primary: var(--color-blue-6);
```

Comment:
We need the overrides for dark mode too. Two options:

1. Rename to `evo-data-viz-light.css` and create the corresponding dark file
2. Move these tokens into `evo-light.css` and `evo-dark.css`

Probably easiest to just go with option 2 for now and we can always split them out later if need be.